uber/site_sections/registration.py

```diff
@@ -492,12 +519,108 @@ def comments(self, session, order='last_name'):
             'attendees': session.query(Attendee).filter(Attendee.comments != '').order_by(order).all()
         }
 
+    def reprint_fee(self, session, attendee_id=None, message='', fee_amount=0, reprint_reason='', refund=None):
```

Comment:
This could go in the AC plugin, but maybe there should just be a printer plugin instead.